TACC/Core-CMS
# feat: djangocms icon plugin support cortal & svg

## Overview

Let users add Cortal Font icons & SVG logos.

## Related

- required by https://github.com/TACC/tup-ui/pull/168/

## Changes

- **added** plugin settings
- **added** plugin icon lists
- **added** plugin templates
- **added** plugin load of https://github.com/TACC/Core-Styles "Cortal" icons

## Testing

0. https://github.com/TACC/Core-CMS/wiki/Test-CMS-Changes
1. Create an "Icon" plugin to a page.
2. Choose any icon from TACC "Cortal" Icons.
3. Save.
4. Verify icon loads.
5. Create an "Icon" plugin to a page.
6. Choose any icon from Logo SVGs.
7. Save.
8. Verify icon loads.

## UI

…

## Notes

### Known Issues

1. Icon previews are blank if editor switches from SVG set to icon set.
    <sup>I suspect a bug with the plugin e.g. https://github.com/django-cms/djangocms-icon/issues/9.</sup>
